site/content/integrate/apps/using-third-party-api/hello-webhooks/_index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Webhooks"
+heading: "Hello, Webhooks!"
+description: "In this example, the http app will demonstrate connect webhooks"
+weight: 90
+---
+
+This is an example of an HTTP app ([source](https://github.com/mattermost/mattermost-plugin-apps/tree/master/examples/go/hello-webhooks)), written in Go and runnable on http://localhost:8080.
+
+- It contains a `manifest.json`, declares itself an HTTP application, requests permissions, and binds itself to locations in the Mattermost user interface.
+- In its `bindings` function it declares two commands: `info` and `send`.
+- The `info` command will post an ephemeral message containing a valid `send` command.
+- The `send` command sends a webhook message to the apps plugin which in turn sends the webhook request to the `Hello, webhooks!` app
+- The hello app receives the webhook, via it's `/webhook` endpoint, and responds with an ephemeral message when the webhook is received.
+
+To install "Hello, Webhooks" on a locally-running instance of Mattermost follow these steps (go 1.16 is required):
+
+```sh
+git clone https://github.com/mattermost/mattermost-plugin-apps.git
+cd mattermost-plugin-apps/examples/go/hello-webhooks
+go run . 
+```
+
+In the Mattermost Desktop client run:
+
+```
+/apps install http http://localhost:8080/manifest.json
+```
+
+### Manifest
+
+`Hello, Webhooks!` is an HTTP app. It requests the *permissions* to act as a System Admin and Bot to access the Mattermost REST API. It also requests permissions to receive 3rd party webhook messages. It binds itself to `/` commands.
+
+```json
+{
+    "app_id": "hello-webhooks",
+    "version": "demo",
+    "display_name": "Hello, Webhooks!",
+    "app_type": "http",
+    "root_url": "http://localhost:8080",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-apps/examples/go/hello-webhooks",
```

Comment:
That's a 404

Reply:
[`95f1d87` (#828)](https://github.com/mattermost/mattermost-developer-documentation/pull/828/commits/95f1d8717f4cc9db0037f0d294f68d32f801c01b)

Reply:
I pointed these to the code base, but wonder if they should point to the relevant developers.mattermost.com section.  

Another idea would be to place a README.md in https://github.com/mattermost/mattermost-plugin-apps/tree/master/examples/go/hello-oauth2 that would inform the user that a better explanation of the example exists in `developers.mattermost.com` and post the link in the README.

Reply:
I'll create a PR to add README files to https://github.com/mattermost/mattermost-plugin-apps/tree/master/examples/go/hello-*